### DISCUSSION TITLE
For some reason Spotube stopped working after version 2.70

### DISCUSSION THREAD

**Discussion by bxb2world** [KRTirtho/spotube#474] [General]
For some reason, Spotube stopped working for me after version 2.7.0. 
I'm using AppImage on Linux Mint 20.3 64bit.
The message I get is as follows: 'GLIBC_2.34' not found. 
What could be happening?

**Comment by gmargriff** [comment]
Seems like Linux Mint 20.* only goes up to glibc_2.31 (you can check that by trying to install it from the repos with `sudo apt install glibc-source`  )

It'll probably return `glibc-source is already the newest version (2.31-0ubuntu9.9).`

Maybe Spotube depends on a newer version of GLIBC (2.34), which brings us to this error message.

You can try:
1. Install an [older spotube version](https://github.com/KRTirtho/spotube/releases/tag/v2.6.0) (as it might work with an older libc, but losing a bunch of new stuff that comes with 2.7.*)
2. Install [Spotube through flatpak ](https://flathub.org/apps/com.github.KRTirtho.Spotube), as it'll probably come with its own builtin libraries **(That's your best bet IMHO)**
3. Upgrade your system to a newer version of Mint (21.*), which can be a good thing in general, but it's not guaranteed that it'll work as I don't know which libc comes with Mint 21
